nginx-directory/files/nginx.conf

```diff
@@ -18,6 +18,7 @@ http {
   resolver     kube-dns.kube-system.svc.%(cluster_dns_suffix)s;
   server {
     listen 80;
+    listen [::]:80;
```

Comment:
I'm not clear what the failure scenario is when ipv6 is not available.

Reply:
there's likely always a link local v6 available